service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/generated/GeneratedOrder.java

```diff
@@ -55,19 +53,19 @@
     private final String childrenDescription;
     private String removalReason;
     private String linkedApplicationId;
+    private String markedFinal;
 
     @JsonIgnore
     public boolean isRemovable() {
-        GeneratedOrderTypeDescriptor descriptor = GeneratedOrderTypeDescriptor.fromType(this.type);
-        return (descriptor.getType() == BLANK_ORDER)
-            || (descriptor.getType() == EMERGENCY_PROTECTION_ORDER)
-            || (descriptor.getType() == CARE_ORDER)
-            || (descriptor.getType() == SUPERVISION_ORDER)
-            || (descriptor.getType() == UPLOAD);
+        return true;
```

Comment:
do we want to keep this method still. It just returns true in all of it's implementations however I can understand the value of keeping it

Reply:
I think as an interface it's good to have it. There's still an SDO that is not removable    @JsonIgnore
    public boolean isRemovable() {
        return isSealed();
    }

Reply:
yeah just seen this one, don't know how I missed it